css/style.css

```diff
@@ -1,6 +1,13 @@
+/* this is for the sales.html */
+
 body {
   font-family: 'Baloo', cursive;
```

Comment:
If you are including a custom font, you need to also include a `link` tag in the HTML to import the font into your page. You can find this font on [Google Fonts](https://fonts.google.com/). Just click the `+`, then copy the embed link from the bar that pops up at the bottom of the screen.